### PR TITLE
[AIRFLOW-4707](https://issues.apache.org/jira/browse/AIRFLOW-4708) tests/plugins made pylint compliant

### PR DESCRIPTION
**Pull request by kik-kik**
Make sure you have checked _all_ steps below.

### Description
- As part of [AIRFLOW-4364 Integrate Pylint](https://issues.apache.org/jira/browse/AIRFLOW-4364)

### Tests
- [ ] no testing needed, only docstring comments added

### CODE REVIEW

**Comment by feluelle** [comment]
Hey @kik-kik, thanks for the PR's. Please include the link in the PR description instead of in the title and prepend your commit message with the ticket number as well. Thanks :)

File: tests/plugins/test_plugin.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""Module containing """
```
Comment (mik-laj) [inline tests/plugins/test_plugin.py:19]: Can you expand this description?

Comment (kik-kik) [reply tests/plugins/test_plugin.py:19]: Hey, sorry I must have gotten distracted in the middle of writing it 🙈 

I will fix it this weekend.

Comment (OmerJog) [reply tests/plugins/test_plugin.py:19]: @kik-kik any progress?

**Pull request closed without merging**